graphql/resolvers/participant/queries.js

```diff
@@ -6,10 +6,9 @@ export default {
     },
 
     participants: async (_, args) => {
-        let regex = new RegExp(args.name);
-
-        return Participant.find({
-            full_name: { $regex: regex, $options: "i" },
+        Object.keys(args).forEach((key) => {
+            args[key] = { $regex: new RegExp(args[key]), $options: "i" };
         });
+        return Participant.find(args);
```

Comment:
If multiple keys are supplied regular expressions are combined with "and"? "or"? It should be obvious from reading the code

Reply:
AND, and it comes from the default behavior of Schema.find(), where it looks for document matching all the specified criteria.
Example for 2 keys:
1. `` args = {full_name: "name", license_no: "345"}``
2. `` args = {full_name: {$regex: \name\, $options: "i"}, license_no: {$regex: \345\, $options: "i"}``
3. ``return Participant.find({``
``    full_name: {$regex: \name\, $options: "i"},``
``    license_no: {$regex: \345\, $options: "i"} ``
`` });``

Reply:
OK, I suggest documenting it somewhere, even in comment.
Beside it - good work, approved.